packages/wallet/src/services/ProviderTracker/TrackedNetworkInfoProviderRequiredByWallet.ts

```diff
@@ -1,30 +1,22 @@
+/* eslint-disable brace-style */
 import { BehaviorSubject } from 'rxjs';
 import { CLEAN_FN_STATS, ProviderFnStats, ProviderTracker } from './ProviderTracker';
-import { NetworkInfoProvider } from '@cardano-sdk/core';
+import { NetworkInfoProviderRequiredByWallet } from '../../types';
 
-export class NetworkInfoProviderStats {
-  readonly healthCheck$ = new BehaviorSubject<ProviderFnStats>(CLEAN_FN_STATS);
-  readonly stake$ = new BehaviorSubject<ProviderFnStats>(CLEAN_FN_STATS);
-  readonly lovelaceSupply$ = new BehaviorSubject<ProviderFnStats>(CLEAN_FN_STATS);
+export class NetworkInfoProviderRequiredByWalletStats {
```

Comment:
```suggestion
export class NetworkInfoRequiredByWalletProviderStats {
```

Reply:
Improved the name in b151972e847831473ed045af83c70d41a8724164